### PR TITLE
[release-1.20] bump gateway-api from 1.0.0-rc2 to 1.0.0

### PR DESCRIPTION
**Pull request by frankbu**
**Please provide a description of this PR:**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module istio.io/istio
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```
Comment (hanxiaop) [inline go.mod:5]: I think this is not needed

Comment (hanxiaop) [reply go.mod:5]: I'll try to fix this so that the 1.0.0 can be included in the rc.0

**Comment by zirain** [comment]
replaced by https://github.com/istio/istio/pull/47699

File: go.mod (hunk 1 above)
Comment (frankbu) [reply go.mod:5]: Thanks.

**Pull request closed without merging**